jOOR-java-8/src/main/java/org/joor/Compile.java

```diff
@@ -50,7 +50,11 @@
  */
 class Compile {
 
-    static Class<?> compile(String className, String content, CompileOptions compileOptions) {
+	static Class<?> compile(String className, String content, CompileOptions compileOptions) {
+		return compile(className, content, compileOptions, false);
+	}
+
+	static Class<?> compile(String className, String content, CompileOptions compileOptions, boolean skipLoaded) {
```

Comment:
Please make sure there is no non-semantic whitespace change, such as this indentation